### ISSUE TITLE
SQLite Database Error when running the program

### ISSUE THREAD

**Issue by lefterisloukas** [alions7000/SEC-EDGAR-text#2]
I installed the `requirements.txt` file and I'm trying to call `SEC-EDGAR-text` from its parent folder.

I am following your instructions but the program shows me an `sqlite` error.

More specifically:

```
File "SEC-EDGAR-text/__main__.py", line 20, in <module>
    from src.control import Downloader
  File "SEC-EDGAR-text/src/control.py", line 11, in <module>
    from .download import EdgarCrawler
  File "SEC-EDGAR-text/src/download.py", line 15, in <module>
    from .utils import args, logger, requests_get
  File "SEC-EDGAR-text/src/utils.py", line 99, in <module>
    sql_connection = sqlite3.connect(db_location)
sqlite3.OperationalError: unable to open database file
```

I saw that you have commented some lines in the source code where you create the DB.

Can you specify the problem? Thanks.

**Comment by RobinDeeCee** [comment]
I got the same problem, Do you already fixed that ?

**Comment by lefterisloukas** [comment]
I didn't find a workaround.

**Comment by RobinDeeCee** [comment]
Workaround:

I'm not a phyton guy:

### SQLITE error
I think this is bescause you're useing windows: so the "/" has to be "\"

```
$ C:\Users\Robin\Desktop>python SEC-EDGAR-text --storage=/tmp/my_storage_folder
$ Enter filings search text (default: 10-K,10-Q): 10-Q
$ Enter start date for filings search (default: 20200319): 20200618
$ Enter end date for filings search (default: 20200917): 20090101
$ Enter filing report period ccyy, ccyymm etc. (default: all periods):
--> /tmp/my_storage_folder
--> /tmp/my_storage_folder\metadata.sqlite3
```


```
$ C:\Users\Robin\Desktop>python SEC-EDGAR-text --storage=\tmp\my_storage_folder
$ Enter filings search text (default: 10-K,10-Q): 10-Q
$ Enter start date for filings search (default: 20200319): 20200618
$ Enter end date for filings search (default: 20200917): 20090101
$ Enter filing report period ccyy, ccyymm etc. (default: all periods):
--> \tmp\my_storage_folder
--> \tmp\my_storage_folder\metadata.sqlite3


```

_This will save everthing in the root map on your pc. You want this data in your project ? use python SEC-EDGAR-text_

### Writing the txt files error
Error:
```
20200618 13:51:41 ERROR Fatal error in company downloading
Traceback (most recent call last):
  File "SEC-EDGAR-text\__main__.py", line 25, in main
    Downloader().download_companies(do_save_full_document=False)
  File "SEC-EDGAR-text\src\control.py", line 96, in download_companies
    seccrawler.download_filings(company_description,
  File "SEC-EDGAR-text\src\download.py", line 75, in download_filings
    log_cache = self.download_filing(filing_metadata, do_save_full_document)
  File "SEC-EDGAR-text\src\download.py", line 248, in download_filing
    sections_log_items = reader_class(
  File "SEC-EDGAR-text\src\document.py", line 38, in get_excerpt
    self.prepare_text()
  File "SEC-EDGAR-text\src\html_document.py", line 50, in prepare_text
    start_time = time.clock()
AttributeError: module 'time' has no attribute 'clock'
```
[From](https://stackoverflow.com/questions/58569361/attributeerror-module-time-has-no-attribute-clock-in-python-3-8)
 > The function time.clock() has been removed, after having been deprecated since Python 3.3: use time.perf_counter() or time.process_time() instead, depending on your requirements, to have well-defined behavior

Fix:
replace time.clock() with time.process_time() in file:

- scr/document.py
- scr/html_document.py

_Still running here, so manybe a other error later today_

**Comment by alions7000** [comment]
New version fixes this for Python 3.8 users. Also add some clarification on the readme, plus 'help' for command line arguments, to help on the other points.
